packages/react-sdk/src/components/Whiteboard/ElementBehaviors/Selection/types.ts

```diff
@@ -25,4 +25,5 @@ export type WithSelectionProps = {
 export type WithExtendedSelectionProps = WithSelectionProps & {
   activeElementIds?: string[];
   overrides?: Elements;
+  enableTextTools?: (isVisible: boolean) => void;
```

Comment:
Maybe I am a bit picky here :D

- `isVisible` should be `enabled`
- Also if you use it, you would expect it to be enabled, if you call the function. Maybe `setTextToolsEnabled` is a better name.